### PR TITLE
PAINTROID-369: Crash NullPointerException: colorpicker.ZoomableImageView

### PR DESCRIPTION
**Pull request by JulianJautz**
display variable in ZoomableImageView could be null; null check added

(https://jira.catrob.at/browse/PAINTROID-369)

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt [hunk 1]
```diff
@@ -312,9 +312,11 @@ class ZoomableImageView :
...
             val point = Point()
             post {
-                display.getSize(point)
-                height = point.y
-                width = point.x
+                display?.let {
+                    it.getSize(point)
+                    height = point.y
+                    width = point.x
+                }
```
Comment (CodeChamp-SS) [inline colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt:319]:
```suggestion
                display?.getSize(point)
                height = point.y
                width = point.x
```
I think you can do the same thing by just adding a `?` 🤔
Also please change the year in the copyright section to 2022 :)
Just noticed that display.getSize() is deprecated in API 30, what do you think about replacing it with something similar to that mentioned in https://stackoverflow.com/questions/63719160/getsize-deprecated-in-api-level-30?

Comment (JulianJautz) [reply colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt]: True, I thought setting height and width should only be done if display is not null.
I added the alternative as described in the thread.

Comment (CodeChamp-SS) [reply colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt]: Thanks for making the changes! Could you please fix the static analysis warnings, then I'll approve the PR :)

Comment (JulianJautz) [reply colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt]: Should be fixed now. I also checked the failing tests, none of them seems to be related to these changes.

Comment (CodeChamp-SS) [reply colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt]: Could you please just check https://jenkins.catrob.at/job/Paintroid/job/PR-1040/6/testReport/junit/org.catrobat.paintroid.test.espresso.dialog/ColorDialogIntegrationTest/Tests___Device_Tests___alphaValueIsSetInSliderWhenChangedInSeekBar/ once locally? rest is fine :)

Comment (JulianJautz) [reply colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt]: It fails locally, but tried it with current develop branch without my changes and it fails too...

Comment (CodeChamp-SS) [reply colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt]: hmm.. then I think this needs to be fixed maybe in a separate PR 🤔, I'll approve this one then.

**Review by CodeChamp-SS — APPROVED** [review]
LGTM!
